examples/kubernetes/dynamic_provisioning_s3/README.md

```diff
@@ -13,12 +13,14 @@ provisioner: fsx.csi.aws.com
 parameters:
   subnetId: subnet-056da83524edbe641
   securityGroupIds: sg-086f61ea73388fb6b
+  autoImportPolicy: NONE
   s3ImportPath: s3://ml-training-data-000
   s3ExportPath: s3://ml-training-data-000/export
   deploymentType: SCRATCH_2
 ```
 * subnetId - the subnet ID that the FSx for Lustre filesystem should be created inside.
 * securityGroupIds - a common separated list of security group IDs that should be attached to the filesystem.
+* autoImportPolicy - the policy FSx will follow that determines how the filesystem is automatically updated with changes made in the linked data repository. For list of acceptable policies, please view the official FSx for Lustre documentation
```

Comment:
I tried looking for the documentation myself but couldn't find it. I found https://docs.aws.amazon.com/fsx/latest/LustreGuide/autoimport-data-repo.html but not the strings that each policy corresponds to. Is it OK to link to documentation here or is it a moving target?

Reply:
I can link to this documentation: https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystemLustreConfiguration.html!

Reply:
Yeah that would be perfect thanks